src/App.tsx

```diff
@@ -1,61 +1,105 @@
-import React from 'react';
-import 'bulma/css/bulma.css';
-import './App.scss';
+import classNames from "classnames";
+import React, { useState } from "react";
+import "bulma/css/bulma.css";
+import "./App.scss";
 
 export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  "Dumplings",
+  "Carrot",
+  "Eggs",
+  "Ice cream",
+  "Apple",
+  "Bread",
+  "Fish",
+  "Honey",
+  "Jam",
+  "Garlic",
 ];
 
+export function getReorderedGoods(
+  goods: string[],
+  sortType: string,
+  isReversed: boolean,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((itemOne: string, itemTwo: string): number => {
+    switch (sortType) {
+      case "ALPHABET":
```

Comment:
Let's use enum for it. As a rule of thumb, it's recommended to avoid using plain strings for switch cases.